src/rgw/rgw_kms.cc

```diff
@@ -383,6 +492,28 @@ static int get_actual_key_from_vault(CephContext *cct,
 }
 
 
+static int get_actual_key_from_kmip(CephContext *cct,
+                                     std::string_view key_id,
+                                     std::string& actual_key)
+{
+  std::string secret_engine = RGW_SSE_KMS_KMIP_SE_KV;
+#if 0
```

Comment:
what's going on in the #if 0 block?

Reply:
cruft; will remove.

I wasn't sure if I needed this as debugging - for now I definitely don't.